test/DatabaseTester.cs

```diff
@@ -121,10 +121,31 @@ public class DatabaseTester {
 			var result = db.ScriptCreate();
 			TestHelper.DropDb("TEST");
 
-			Assert.That(result, Is.StringContaining("CREATE  NONCLUSTERED INDEX [MyIndex] ON [dbo].[MyTable] ([Id]) WHERE ([EndDate] IS NULL)"));
+			Assert.That(result, Is.StringContaining("CREATE  NONCLUSTERED INDEX [MyIndex] ON [dbo].[MyTable] ([Id] ASC) WHERE ([EndDate] IS NULL)"));
 		}
 
-		[Test]
+        [Test]
```

Comment:
Yup, this test broke after we merged his master back into our code since somebody else added support for ASC/DESC